inc/commondbchild.class.php

```diff
@@ -881,4 +881,25 @@ public static final function getItemField($itemtype): string {
 
       throw new \RuntimeException('Cannot guess field for itemtype ' . $itemtype . ' on ' . static::class);
    }
+
+   protected function autoinventoryInformation() {
```

Comment:
I may miss something, but why this method is located in `CommonDBChild` and not in `Inventoriable` trait ?

Reply:
inventoriable trait is used on main inventory items only (computer, printer, phone, networkequipment)